pkg/virt-controller/services/template.go

```diff
@@ -666,6 +666,9 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, i
 	}
 
 	pod.Spec.Volumes = append(pod.Spec.Volumes, sidecarVolumes...)
+	if needExtraVirtiofs(vmi) {
+		pod.Spec.Volumes = append(pod.Spec.Volumes, *extraVirtiofsVolume())
+	}
```

Comment:
ditto, why this is necessary

Reply:
yes, because it is an additional empty dir where virt-handler locate the placeholder socket

Reply:
Added comment why we need this in the function declaration